Doc/library/typing.rst

```diff
@@ -27,12 +27,13 @@ This module provides runtime support for type hints.
 
 Consider the function below::
 
-   def moon_weight(earth_weight: float) -> str:
-       return f'On the moon, you would weigh {earth_weight * 0.166} kilograms.'
+   def distance(kilometers: float) -> str:
+       return f"The distance is {kilometers} kilometers ({kilometers / 1.60934} miles)."
```

Comment:
[Sorry for being picky] This gets around the issue with mass/weight unit confusion. But it feels like a worse example otherwise, because it's not self-evident from the name of the function anymore what exactly the function does or in what situation you'd use it -- the name `distance` feels kinda generic. Whereas `moon_weight`, you know exactly when you'd use it (when you want to find out how much you'd weigh on the moon).

Can we improve the name of the function? Or think of a different example where the function seems obviously useful for some purpose?

What about something like

```suggestion
   def surface_area_of_cube(edge_length: float) -> str:
       return f"The surface area of the cube is {6 * edge_length ** 2}"
```

Reply:
May I boldly assume that your first suggestion would have been the circle but then you thought about the needs of importing `math.pi`? (Otherwise, I'm happy with your suggestion, so sorry Jelle, but the credits go to Alex now!)

Reply:
I also added the link to the term `type hint` in the glossary (I think it's nice to have it but tell me if it's too much).

Reply:
> May I boldly assume that your first suggestion would have been the circle but then you thought about the needs of importing `math.pi`?

No comment :P

Reply:
(My second thought suggestion would have been "area of a triangle", but that requires two parameters in the function signature 😆)

Reply:
One of my suggestion would have been the conversion hours / seconds but it would have been less generic (it's interesting how easily we can come up with examples but how hard it is to find good ones!)